modules/accredited_representative_portal/app/controllers/accredited_representative_portal/application_controller.rb

```diff
@@ -1,29 +1,24 @@
 # frozen_string_literal: true
 
 module AccreditedRepresentativePortal
-  class ApplicationController < ::ApplicationController
+  class ApplicationController < SignIn::ApplicationController
+    include SignIn::AudienceValidator
+    include Authenticable
     # TODO: Add ARP to Datadog Service Catalog #77004
     #   https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/77004
     # It will be the dd-service property for your application here:
     #   https://github.com/department-of-veterans-affairs/vets-api/tree/master/datadog-service-catalog
-    service_tag 'accredited-representative-portal'
+    service_tag 'accredited-representatives'
```

Comment:
Is there a reason the service tag name was changed?

Reply:
oof yeah good catch, this is my bad. they changed `accredited-representatives` to `accredited-representative-portal` and I had to fix a bunch of conflicts and must have messed this one up